### PR TITLE
Script to disable CI builds without .shoov.yml - Fix

### LINKED ISSUE #321: Script to disable CI builds without .shoov.yml
Script should go over all CI builds (or above a certain ID), check if there's a `.shoov.yml`, and if not - disable the CI build.

### PR DESCRIPTION
**Pull request by HelenaEksler**
#321

https://github.com/shoov/shoov/issues/321#issuecomment-166273486

### CODE REVIEW

**Comment by HelenaEksler** [comment]
@amitaibu Fixed.
selection_070

File: shoov/modules/custom/shoov_ci_build/drush/shoov_ci_build.drush.inc [hunk 1]
```diff
@@ -172,7 +172,8 @@ function drush_shoov_ci_build_disable_ci_builds() {
     variable_set('shoov_ci_build_last_checked_for_config_file', 0);
     return;
   }
-  $nodes = node_load_multiple(array_keys($result['node']));
+  $ids = array_keys($result['node']);
```
Comment (amitaibu) [inline shoov/modules/custom/shoov_ci_build/drush/shoov_ci_build.drush.inc:175]: how is this fixing it?

Comment (HelenaEksler) [inline shoov/modules/custom/shoov_ci_build/drush/shoov_ci_build.drush.inc:175]: `$ids` variable wasn't defined. Now it is.

Comment (amitaibu) [inline shoov/modules/custom/shoov_ci_build/drush/shoov_ci_build.drush.inc:175]: Oh, now I see - thanks.

**Pull request merged**